### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SashaViun**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
```
Comment (vanst1) [inline src/calculateRentalCost.js:33]: [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 5

**Review by vanst1 — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 1 above)
Comment (SashaViun) [reply src/calculateRentalCost.js:33]: got it

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,19 @@
...
+
+  if (days < 3) {
+    return days * cost;
+  }
+
+  if (days < 7) {
+    return (days * cost) - smallDiscount;
+  }
+
+  return (days * cost) - bigDiscount;
```
Comment (alexandra-protyanova) [inline src/calculateRentalCost.js:43]: Make sense to make variable for this operation `(days * cost)`, to prevent code duplicating

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — APPROVED** [review]
Well done 👍

**Pull request closed without merging**